frontend/src/components/ToolTip/ToolTip.tsx

```diff
@@ -0,0 +1,28 @@
+import { FunctionComponent } from 'react';
+import { ToolTipText, ToolTip } from './ToolTip.style';
+import { colorPalette } from 'stylesheet';
+
+interface Props {
+  children?: any;
+  toolTipText?: string;
+  bottom ?: string;
+  color ?: string;
+  backgroundColor ?: string;
+  reverse ?: boolean;
+}
+
+const ToolTipGT: FunctionComponent<Props> = ({ children,
+                                                toolTipText,
+                                                bottom="115%",
+                                                color = colorPalette.primary1,
+                                                backgroundColor = colorPalette.primary2,
+                                                reverse = false }) => (
+  <ToolTip>
```

Comment:
This review is for accessibility purposes to match the https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Tooltip_Role specification ⬇️ 

First step, it would be nice to add `role="tooltip"` to `<Tooltip />`.

And then add an optional prop parameter `id? : string`. If set, display the `id` attribute to `<Tooltip />` and the `aria-describedby` attribute to `<TooltipText />`. 

And it would be better to create a generative `id` if the related property is not defined 🙏